models/api.js

```diff
@@ -0,0 +1,7 @@
+const { readFile } = require("fs/promises");
+
+exports.readJSON = () => {
+  return readFile("./endpoints.json", "utf-8").then((file) => {
+    return file;
+  });
+};

```

Comment:
Just like in the test you dont have to readFile a json, you can require it directly as a variable and it will work like any normal JS object